build/efcore-scenarios.yml

```diff
@@ -42,14 +42,18 @@ parameters:
       arguments: $(efcoreJobs) --benchmarkdotnet:*RawSqlQuerySqliteTests* -n Sqlite --project-file benchmark/EFCore.Sqlite.Benchmarks/EFCore.Sqlite.Benchmarks.csproj
     - displayName: SimpleQuerySqliteTests
       arguments: $(efcoreJobs) --benchmarkdotnet:*SimpleQuerySqliteTests* -n Sqlite --project-file benchmark/EFCore.Sqlite.Benchmarks/EFCore.Sqlite.Benchmarks.csproj
-    - displayName: Update
+    - displayName: SimpleQuerySqliteTests
+      arguments: $(efcoreJobs) --benchmarkdotnet:*NavigationsQuerySqliteTests* -n Sqlite --project-file benchmark/EFCore.Sqlite.Benchmarks/EFCore.Sqlite.Benchmarks.csproj
+    - displayName: NavigationsQuerySqliteTests
+      arguments: $(efcoreJobs) --benchmarkdotnet:*QueryCompilationSqliteTests* -n Sqlite --project-file benchmark/EFCore.Sqlite.Benchmarks/EFCore.Sqlite.Benchmarks.csproj
+    - displayName: QueryCompilationSqliteTests
```

Comment:
Looks like a mistake - `displayName: Update` is gone no?

Reply:
Correct